src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,18 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/Todo';
+
+type Todos = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Todos> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => {
+      return (
+        <TodoInfo todo={todo} key={todo.id} />
+      );
+    })}
```

Comment:
just use arrow fucntion
```suggestion
    {todos.map(todo => (
      <TodoInfo todo={todo} key={todo.id} />
    ))}
```